internal/handlers/tigris/tigrisdb/query_test.go

```diff
@@ -0,0 +1,242 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tigrisdb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tigrisdata/tigris-client-go/config"
+	"github.com/tigrisdata/tigris-client-go/driver"
+	"go.uber.org/zap"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/iterator"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+)
+
+func TestQueryDocuments(t *testing.T) {
+	t.Parallel()
+
+	t.Run("QueryDocuments", func(t *testing.T) {
+		t.Parallel()
+
+		dbName, collName, ctx, tdb := setup(t)
+
+		var inserted []*types.Document
+		for i := 0; i < 10; i++ {
+			doc := must.NotFail(types.NewDocument("_id", int64(i)))
+			err := tdb.InsertDocument(ctx, dbName, collName, doc)
+			require.NoError(t, err)
+
+			inserted = append(inserted, doc)
+		}
+
+		iter, err := tdb.QueryDocuments(ctx, &FetchParam{
+			DB:         dbName,
+			Collection: collName,
+		})
+		require.NoError(t, err)
+
+		require.NotNil(t, iter)
+
+		defer iter.Close()
+
+		var queried []*types.Document
+
+		i := 0
+		for {
+			var n int
+			var doc *types.Document
+
+			n, doc, err = iter.Next()
+			if err != nil {
+				if errors.Is(err, iterator.ErrIteratorDone) {
+					break
+				}
+
+				require.NoError(t, err)
+			}
```

Comment:
```suggestion
			if errors.Is(err, iterator.ErrIteratorDone) {
					break
			}
```

I think single if should work here :hugs: The error is check next line.